packages/react-table/src/deprecated/components/Table/__tests__/__snapshots__/Table.test.tsx.snap

```diff
@@ -1602,11 +1602,10 @@ exports[`Table Collapsible nested table 1`] = `
         </td>
       </tr>
       <tr
-        class="pf-v6-c-table__tr pf-v6-c-table__expandable-row"
```

Comment:
Deprecated table tests/examples looks like it needs a similar update potentially, but I think that's probably fine in a follow up or until someone opens an issue for deprecated specifically.